Web3/README.md

```diff
@@ -7,7 +7,9 @@
 - [Blockchain Technologies](#blockchain-technologies)<br>
 - [News](#news)<br>
 - [Community](#community)<br>
-- [Youtube Channels](#youtube-channels)
+- [Youtube Channels](#youtube-channels)<br>
+- [Decentralized Finance (DeFi) Projects](#Decentralized-Finance-(DeFi)-Projects)
```

Comment:
```suggestion
- [Decentralized Finance (DeFi)](#Decentralized-Finance-(DeFi))
```

Reply:
I have made the proposed change, please review and let me know if any changes are to be made.